src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/M2FiltersArgumentCollection.java

```diff
@@ -119,6 +121,12 @@
     @Argument(fullName = UNIQUE_ALT_READ_COUNT_LONG_NAME, shortName = "unique", optional = true, doc = "Filter a variant if a site contains fewer than this many unique (i.e. deduplicated) reads supporting the alternate allele")
     public int uniqueAltReadCount = 0;
 
+    @Argument(fullName = N_RATIO_LONG_NAME, optional = true, doc = "Filter a variant if the ratio of Ns in the pileup is greater or equal to this value. A ratio of 0 will not apply filter.")
```

Comment:
"Ratio of 0 will not apply the filter" - Is this true?

Reply:
yeah that's not true anymore, a ratio of 0 will apply the filter to everything 
I'm just going to remove this